app/about_us/OurPartners.js

```diff
@@ -0,0 +1,24 @@
+const OurPartners = () => {
+    const partners = [
+      { name: "TechCorp", logo: "/partners/techcorp.png" },
```

Comment:
Are you sure this path exists? I cannot find the "partner" folder inside the public folder